package.json

```diff
@@ -76,5 +76,8 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "dependencies": {
+    "sinon": "^7.5.0"
```

Comment:
Was jest unable to provide a mock that would allow you to test the parts that you wanted to cover?

Reply:
sinon's way of stubbing async function is pretty straight forward. I thought adding it would give options to developers. But I'll remove, it I saw a way of doing it using jest also.

Reply:
removed